aioprocessing/managers.py

```diff
@@ -1,29 +1,22 @@
 import asyncio
 from multiprocessing.util import register_after_fork
 from queue import Queue
-from threading import (
```

Comment:
These imports should still come from `threading`. They're getting proxied, so we don't need the multiprocessing versions.